build.gradle

```diff
@@ -132,11 +135,21 @@ sourceSets.main.java.srcDir "${openapiTargetDir}/src/main/java"
 ideaModule.dependsOn swaggerSources.server.code
 // end of OpenAPI Server Generation
 
+def googleCredentialsFile = "${projectDir}/src/test/resources/rendered/sa-account.json"
```

Comment:
nit: add a comment `// This is the path to the default Google service account for the janitor service to run as. `